sematic/examples/cifar_classifier/pipeline.py

```diff
@@ -0,0 +1,106 @@
+# Standard Library
+from dataclasses import dataclass
+
+# Third-party
+from ray.train.torch.torch_checkpoint import TorchCheckpoint
+
+# Sematic
+import sematic
+from sematic.ee.ray import RayCluster, SimpleRayCluster
+from sematic.examples.cifar_classifier.train_eval import (
+    EvaluationConfig,
+    EvaluationResults,
+    TrainingConfig,
+    evaluate_classifier,
+    train_classifier,
+)
+
+
+@dataclass
+class PipelineResults:
+    final_checkpoint: TorchCheckpoint
+    evaluation_results: EvaluationResults
+
+
+@sematic.func(inline=False)
+def train(config: TrainingConfig) -> TorchCheckpoint:
+    """# Train an image classifier on the CIFAR10 dataset
+
+    ## Inputs
+    - **config**: configuration for the compute and hyperparameters to be
+    used for training
+
+    ## Returns
+    The last checkpoint produced during training
+    """
+    cluster_config = SimpleRayCluster(
+        n_nodes=config.n_workers, node_config=config.worker
+    )
+    with RayCluster(config=cluster_config):
+        return train_classifier(config)
+
+
+@sematic.func(inline=False)
+def evaluate(
+    checkpoint: TorchCheckpoint, config: EvaluationConfig
+) -> EvaluationResults:
+    """# Run evaluation data on an image classification model
+
+    ## Inputs
+    - **checkpoint**: The checkpoint from training containing the model parameters to
+    be evaluated
+    - **config**: The configuration for the compute to be used during evaluation
+
+    ## Returns
+    Some basic statistics on the evaluation, and a confusion matrix plot for the
+    evaluation.
+    """
+    cluster_config = SimpleRayCluster(
+        n_nodes=config.n_workers, node_config=config.worker
+    )
+    with RayCluster(config=cluster_config):
+        return evaluate_classifier(checkpoint, config)
+
+
+@sematic.func(inline=True)
+def bundle_results(
+    final_checkpoint: TorchCheckpoint, evaluation_results: EvaluationResults
+) -> PipelineResults:
+    """# Combine the results into a single data structure.
```

Comment:
```suggestion
    """# Combine the results into a single data structure
```

Reply:
this is a section header, which usually don't have periods

Reply:
Yes, the suggestion is to remove the period. Please do so everywhere.

Reply:
done